content/automate/ManageIQ/System/Event/EmsEvent/Azure.class/virtualmachines_poweroff_endrequest.yaml

```diff
@@ -0,0 +1,14 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: virtualMachines_powerOff_EndRequest
+    inherits: 
+    description: 
+  fields:
+  - rel4:
+      value: "/System/event_handlers/event_action_refresh?target=ems"
+  - rel5:
+      value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_suspend&param="
```

Comment:
@gmcculloug @bronaghs 
So a ```power_off``` policy event would be raised for this event and the Suspend task from UI should be removed. Is it correct?